src/einsteinpy/metric/kerr.py

```diff
@@ -193,14 +194,15 @@ def calculate_trajectory(
             t_bound=end_lambda,
             **OdeMethodKwargs
         )
-        _scr = self.schwarzschild_r.value * 1.001
+      # _scr = self.schwarzschild_r.value * 1.001
+        _kerr_r = self.kerr_r.value * 1.001
```

Comment:
Can you name call it like `_event_hor = kerr_utils.event_horizon(self.schwarzschild_r.value, self.a) * 1.001

Reply:
yeah I should have done this!